quill/pki/load/private_key.go

```diff
@@ -10,10 +10,10 @@ import (
 	"github.com/anchore/quill/internal/log"
 )
 
-func loadPrivateKey(path string, password string) (crypto.PrivateKey, error) {
+func NewPrivateKey(path string, password string) (crypto.PrivateKey, error) {
```

Comment:
similar comment is this really "New"?

Reply:
I went ahead and made the same adjustments to the remaining helper functions in this package